server/sql/migrations/00005_add_column_uuid_to_users_and_subscriptions.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE users
+  ADD user_id UUID NOT NULL DEFAULT gen_random_uuid();
```

Comment:
we'll probably want to having the naming match the value here; I think the column should just be `user_uuid` and we can keep the `user_id` as the BIGSERIAL number

Reply:
This way we can use `user_id` internally and surface `user_uuid` to the client